lib/streamlit/elements/radio.py

```diff
@@ -79,6 +80,8 @@ def radio(
             An optional boolean, which disables the radio button if set to
             True. The default is False. This argument can only be supplied by
             keyword.
+        horizontal : bool
+            An optional boolean, which makes radio group horizontal
```

Comment:
Nit: Perhaps this could be a little more descriptive? Like:
"An optional boolean, which orients the radio group horizontally. The default is false (vertical buttons). This argument can only be supplied by keyword."